src/test/kotlin/org/jitsi/nlj/test_utils/RtpPacketGenerator.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright @ 2018 - Present, 8x8 Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj.test_utils
+
+import com.nhaarman.mockitokotlin2.spy
+import org.jitsi.nlj.PacketInfo
+import org.jitsi.nlj.util.Bandwidth
+import org.jitsi.rtp.rtp.RtpPacket
+import java.time.Duration
+
+class RtpPacketGenerator internal constructor(
+    /**
+     * Length in bytes of each packet.
+     */
+    val lengthBytes: Int,
+    /**
+     * The number of packets to generate.
+     */
+    val count: Int,
+    /**
+     * The interval to advance the clock after each packet.
+     */
+    val interval: Duration,
+    val clock: FakeClock = spy()
```

Comment:
`FakeClock` doesn't need to be instantiated with `spy()` anymore, it's an all grown up class now.